book/02-git-basics/sections/viewing-history.asc

```diff
@@ -127,14 +127,14 @@ Date:   Sat Mar 15 10:31:28 2008 -0700
  3 files changed, 54 insertions(+)
 ----
 
-As you can see, the `--stat` option prints below each commit entry a list of modified files, how many files were changed, and how many lines in those files were added and removed.
-It also puts a summary of the information at the end.
+Tulad ng iyong nakikita, ang `--stat` ang opsyon ay nagpapakita sa ibabaw sa bawat commit na entry sa listahan ng mga binagong mga file, kung gaano karaming mga file ang binago, at gaano karami ang mga linya sa mga file ang idinagdag at inalis.
+Naglalagay din ito ng buod ng impormasyon sa dulo.
 
-Another really useful option is `--pretty`.
-This option changes the log output to formats other than the default.
-A few prebuilt options are available for you to use.
-The `oneline` option prints each commit on a single line, which is useful if you're looking at a lot of commits.
-In addition, the `short`, `full`, and `fuller` options show the output in roughly the same format but with less or more information, respectively:
+Ang isa pang talagang kapaki-pakinabang na opsyon ay ang `--pretty`.
+Ang opsyon na ito ay nagbabago ng log output para i-format maliban sa default.
+Ang ilang prebuilt na mga opsyon ay magagamit para sa iyo upang gamitin.
```

Comment:
@robin-ho , can you recheck the source please and update accordingly?
The sentence does not make sense. :)

Reply:
@severinolorillajr will check on it.

Reply:
Original sentence : A few prebuilt options are available for you to use.
Change translation to : Ang kaunting prebuilt na mga opsyon ay magagamit para gamitin mo.